wordpress/wp-content/plugins/cds-base/classes/Modules/Contact/ContactForm.php

```diff
@@ -99,49 +201,64 @@ class="gc-textarea"
                     ></textarea>
                     <div class="buttons">
                         <button class="gc-button gc-button" type="submit" id="submit">
-                            <?php _e("Submit", "cds-snc"); ?>
+                            <?php _e('Submit', 'cds-snc'); ?>
                         </button>
                     </div>
                 </form>
+
+                <?php endif; ?>
                 <?php
             } else {
-                $current_url = home_url(add_query_arg(array(), $wp->request));
-                ?>
+                $current_url = home_url(add_query_arg([], $wp->request)); ?>
                 <form id="contact-form-step-1" method="POST" action="<?php echo $current_url; ?>">
                     <div role="group" aria-labelledby="contact_types">
                         <label for="contact_types">
-                            <span id="contact_types" class="hidden"><?php _e("Contact Types", "cds-snc") ?></span>
+                            <span id="contact_types" class="hidden"><?php _e('Contact Types', 'cds-snc'); ?></span>
                         </label>
                         <div class="focus-group" style="margin-bottom: 1.75rem">
-                            <?php $this->radioField("contact-type", "request-site", __("Request a site", "cds-snc")); ?>
                             <?php $this->radioField(
```

Comment:
It's a little bit odd that there is initially no form heading, and then one shows up. 

eg, here's me clicking "request a site"

| before | after |
|--------|-------|
|  no heading above the form, so I added `Contact form` as an `<h3>`       |  now we have `Contact form` as a `<h3>` and `Request a site` as an `<h2>`      |
|   <img width="750" alt="Screen Shot 2021-12-07 at 09 55 39" src="https://user-images.githubusercontent.com/2454380/145055540-97437593-6fa0-4d5f-800f-421c0cc59e59.png">    |   <img width="768" alt="Screen Shot 2021-12-07 at 09 56 27" src="https://user-images.githubusercontent.com/2454380/145055545-f2785631-cc64-402b-98de-8fbff5f031a4.png">    |


If we put this on a page with no other info -- just and `h1` and this contact form -- then it will be fine. But if it's halfway down the page, it might be a weird experience.